testem.js

```diff
@@ -0,0 +1,94 @@
+// Adapted from node ./node_modules/istanbul/lib/cli.js report
+"use strict";
+var path = require("path");
+var http = require("http");
+var fs = require("fs");
+var shell = require("shelljs");
+var port = 7358;
+var server;
+
+module.exports = {
+    serve_files: [
+        "instrumented/*.js"
+    ],
+    "framework": "qunit",
+    "test_page": [
+        "tests/static/tests-binder-array.html",
+        "tests/static/tests-binder-checkbox.html",
+        "tests/static/tests-binder-clear.html",
+        "tests/static/tests-binder-encoding.html",
+        "tests/static/tests-binder-long.html",
+        "tests/static/tests-binder-radio.html",
+        "tests/static/tests-binder-select.html",
+        "tests/static/tests-binder-short.html",
+        "tests/static/tests-binder-textarea.html"
+    ],
+    "parallel": 5,
+    src_files: [
+        "src/*.js"
+    ],
+
+    // proxy to the create http server
+    // phantomjs has problems with POSTs to different port
+    "proxies": {
+        "/coverage": {
+            "target": "http://localhost:7358"
+        }
+    },
+
+    // instrument files, spin up http server to write coverage data to disk
+    on_start: function (config, data, callback) {
```

Comment:
Comments needed here to explain how this bulky file will be factored. Preferably even before this becomes mature we should store the variable material (the list of files) and the constant material (these functions) in separate files.